src/createElement.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import StemContainer from './StemContainer';
```

Comment:
p.s. if we introduce generic sounding components like that, I think it might be nice to prefix the displayName. Otherwise it might be confusing to know if a `RootContainer` came from us or Relay.